deploy/buildtools/buildafi.py

```diff
@@ -21,6 +24,32 @@ def get_deploy_dir():
         deploydir = local("pwd", capture=True)
     return deploydir
 
+def sim_local(cmdline, capture=False):
+    """ setup environment and locally execute a cmdline in sim subdir"""
+    ddir = get_deploy_dir()
+    with prefix('cd ' + ddir + '/../'), \
+         prefix('export RISCV={}'.format(os.getenv('RISCV', ""))), \
+         prefix('export PATH={}'.format(os.getenv('PATH', ""))), \
+         prefix('export LD_LIBRARY_PATH={}'.format(os.getenv('LD_LIBRARY_PATH', ""))), \
+         prefix('source ./sourceme-f1-manager.sh'), \
+         prefix('cd sim/'), \
+         InfoStreamLogger('stdout'), \
+         InfoStreamLogger('stderr'):
+        return local(cmdline, capture=capture)
+
+def get_sim_makevar(buildconfig, varname):
```

Comment:
This feels to me like it belongs in a more general place since it's probably more widely useful.

Reply:
I'll take a look for a better place.

Reply:
@davidbiancolin do you have thoughts on where else it would be useful to have this?  

I thought about moving this stuff into `buildtools/buildconfig.py` but `buildconfig` doesn't have any fabric stuff in it and it seemed good to leave fabric out of the buildconfig.

Perhaps as a new module in the `util` package?  Maybe `util/simshim.py`?  Or, maybe `util/simmakefiles.py`?

If we start moving `get_sim_makevar`, it probably makes sense to put `sim_local` in the new place.  What about `buildafi.get_deploy_dir()`?  Should we move that too?  While we're talking about get_deploy_dir, it makes more sense to me that the GlobalBuildConfig and RunTimeConfig would capture the working directory and pass it down to be used rather than introspected multiple times by ssh'ing back from all build hosts to the manager to run `pwd`. 

I'm starting to feel like moving this somewhere else is part of a larger refactor of this code.  Let me know what you think.

Reply:
I think you're probably right. We can punt on that, there's enough going on in this PR.